cmake/TestSetup.cmake

```diff
@@ -25,9 +25,12 @@ if(Python3_Interpreter_FOUND)
     message(FATAL_ERROR "Python version requirements not satisfied for running regression tests.")
   endif()
 endif()
-# Ensure all required packages are present
-include(${PROJECT_SOURCE_DIR}/cmake/PythonModuleCheck.cmake)
-required_python_modules_found("${REQUIRED_PYTHON_MODULES}")
+
+if (PARTHENON_ENABLE_PYTHON_MODULE_CHECK)
+  # Ensure all required packages are present
+  include(${PROJECT_SOURCE_DIR}/cmake/PythonModuleCheck.cmake)
```

Comment:
Can you change PROJECT_SOURCE_DIR to PARTHENON_SOURCE_DIR?

Reply:
Done.